external-crates/move/move-execution/v0/crates/move-vm-runtime/src/loader.rs

```diff
@@ -1700,7 +1694,7 @@ pub(crate) struct Function {
     type_parameters: Vec<AbilitySet>,
     native: Option<NativeFunction>,
     def_is_native: bool,
-    scope: Scope,
+    scope: ModuleId,
```

Comment:
Nit (and maybe you do this in commits above this): `scope` seems like a possibly less-ideal name for it now? Maybe `module` or `owning_module` (or something else like that)?